README.md

```diff
@@ -1,45 +1,39 @@
-This project was bootstrapped with [Create React Native App](https://github.com/react-community/create-react-native-app).
+# Annotated IPG mobile app
 
-Below you'll find information about performing common tasks. The most recent version of this guide is available [here](https://github.com/react-community/create-react-native-app/blob/master/react-native-scripts/template/README.md).
+This app is created to help the consultation of the [AIPG](https://blogs.magicjudges.org/rules/ipg/#annotated-ipg) in every situation due to his mobile nature. It contains offline version of pages, with internal hyper-links and a full-text search. Card's images need network to be shown.
 
-## Table of Contents
+It is released both for [Android](https://play.google.com/store/apps/details?id=net.italianmagicjudges.aipg) and [iOS](https://itunes.apple.com/us/app/annotated-ipg/id1449453573?mt=8&ign-mpt=uo%3D4).
 
-* [Updating to New Releases](#updating-to-new-releases)
-* [Available Scripts](#available-scripts)
-  * [npm start](#npm-start)
-  * [npm test](#npm-test)
-  * [npm run ios](#npm-run-ios)
-  * [npm run android](#npm-run-android)
-  * [npm run eject](#npm-run-eject)
-* [Writing and Running Tests](#writing-and-running-tests)
-* [Environment Variables](#environment-variables)
-  * [Configuring Packager IP Address](#configuring-packager-ip-address)
-* [Adding Flow](#adding-flow)
-* [Customizing App Display Name and Icon](#customizing-app-display-name-and-icon)
-* [Sharing and Deployment](#sharing-and-deployment)
-  * [Publishing to Expo's React Native Community](#publishing-to-expos-react-native-community)
-  * [Building an Expo "standalone" app](#building-an-expo-standalone-app)
-  * [Ejecting from Create React Native App](#ejecting-from-create-react-native-app)
-    * [Build Dependencies (Xcode & Android Studio)](#build-dependencies-xcode-android-studio)
-    * [Should I Use ExpoKit?](#should-i-use-expokit)
-* [Troubleshooting](#troubleshooting)
-  * [Networking](#networking)
-  * [iOS Simulator won't open](#ios-simulator-wont-open)
-  * [QR Code does not scan](#qr-code-does-not-scan)
+This app is proudly started by [IMJ Community](http://italianmagicjudges.net/):
+- Project leader [Andrea Sciarrotta](https://apps.magicjudges.org/judges/Hawanaghana/)
+- Core developer [Piero Nicolli](https://apps.magicjudges.org/judges/Private_BigHead/)
+- Core developer [Matteo Manchi](https://apps.magicjudges.org/judges/Takeno/)
```

Comment:
I would put your name first, and simply keep it in alphabetical order (also, you did most of the work :P but I think alphabetical makes sense)

Reply:
I'm fine with this order :)